.azure-pipelines/main.yml

```diff
@@ -4,9 +4,9 @@ jobs:
     vmImage: VS2017-Win2016
   steps:
   - template: common/build.yml
+  - template: common/webpack-prod.yml # This is required since we're not running publish
```

Comment:
I would prefer if 'webpack-prod' were a part of `common/build.yml` for two reasons:
1. Then it's run for every OS and you don't need separate windows/mac steps
1. I think it works better as a separate step than as a part of `vscode:prepublish` (as currently specified in package.json). I've had issues where commands run as a part of `prepublish` don't output errors

Reply:
Been wanting to talk to you in person about this.  There's kind of a conflict between what's needed for command-line to work and the Azure pipelines, since they don't share dependency management.  E.g. I think "vsce package" on command-line for a dev should automatically run build and webpack (and vscode:prepublish is the standard mechanism for that), but then if I also have a separate step for it in Azure, it will get run twice, increasing the build times.

Reply:
At the moment I've been more interested in getting it working, but I do want it optimized.